src/main/java/org/alliancegenome/curation_api/model/ingest/dto/fms/GeneExpressionFmsDTO.java

```diff
@@ -0,0 +1,19 @@
+package org.alliancegenome.curation_api.model.ingest.dto.fms;
+
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import org.alliancegenome.curation_api.model.ingest.dto.base.BaseDTO;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class GeneExpressionFmsDTO extends BaseDTO {
+	private String geneId;
+	private String assay;
+	private String dateAssigned;
+	private PublicationFmsDTO evidence;
+	private WhenExpressedDTO whenExpressed;
+	private WhereExpressedDTO whereExpressed;
+
```

Comment:
Looks like this class is missing the `crossReference` maybe there isn't any for expression?

Reply:
Not in LinkML

Reply:
@chris-grove the json schema does have a `crossReference` but the `GeneExpressionAnnotation` in LinkML lists a: `crossReference:                                                     ExpressionAnnotation.data_provider.cross_reference` do we need these or not?

Reply:
Let me correct: at least one MOD, FB, does not populate the field, but it is there in LinkML. That's why I confused!

Reply:
its not a required field but it needs to be captured

Reply:
@oblodgett @abecerra 
In the FMS, data provider and corresponding cross reference are really only included in the meta data for FMS file submissions and only refer to, for example, "WB" and "homepage", with no particular curie for the individual annotations. So, this is the only place in the FMS-submitted data to infer what the data provider is, but it is not a ID/cross-reference for an individual annotation. Nobody submits those.